app/forms/declaration/ProcedureCodes.scala

```diff
@@ -32,6 +32,9 @@ object ProcedureCodes {
   private val procedureCodeLength = 4
   private val additionalProcedureCodeLength = 3
 
+  def extractProcedureCode(form: ProcedureCodes): (Option[String], Option[String]) =
```

Comment:
Could this not live on the `ProcedureCodes` case class? Then it would just be used like `instance.extractProcedureCode` rather than `ProcedureCodes.extractprocedureCode(instance)`

Reply:
ok will change